CHANGES_NEXT_RELEASE

```diff
@@ -1,3 +1,4 @@
+- Include Large refactor of IOTA Lib code to make it simpler
```

Comment:
It seems that only test are being modified, so this entry is not needed.

Note that when we release the next IOTA-UL, an entry like this will be included:

```
- Upgrede iotagent-node-lib version from X to Y
```

which covers the IOTA refactor.